### ISSUE TITLE
Gulp Serve receives no request from SharePoint tenant

### ISSUE THREAD

**Issue by Ziefle-T** [SharePoint/sp-dev-docs#8722]
### What type of issue is this?

Question

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### Target SharePoint environment

SharePoint Online

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [ ] 💥 Microsoft Edge
- [ ] 💥 Google Chrome
- [X] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

- firefox 102.7.0esr (64-bit)
- 1.15.2 (SPFX Version)
- v16.16.0 (Node JS)

### Issue description

I followed [this](https://www.youtube.com/watch?v=6WTtjXP5yW4) tutorial to set up my SPFX environment. Running "gulp serve" i receive the following output:Build target: DEBUG
[14:24:05] Using gulpfile C:\...\gulpfile.js
[14:24:05] Starting 'serve'...
[14:24:05] Starting gulp
[14:24:05] Starting subtask 'spfx-serve'...
[14:24:05] [spfx-serve] To load your scripts, use this query string: ?debug=true&noredir=true&debugManifestsFile=https://localhost:4231/temp/manifests.js
[14:24:11] Starting server...
[14:24:11] Finished subtask 'spfx-serve' after 5.47 s
[14:24:11] Starting subtask 'pre-copy'...
[14:24:13] Finished subtask 'pre-copy' after 2.28 s
[14:24:13] Starting subtask 'copy-static-assets'...
[14:24:13] Starting subtask 'sass'...
[14:24:13] Server started https://localhost:4231
[14:24:13] LiveReload started on port 35729
[14:24:13] Running server
[14:24:13] Opening https://.../.../_layouts/workbench.aspx using the default OS app
[14:24:14] Finished subtask 'sass' after 871 ms
[14:24:14] Starting subtask 'lint'...
[14:24:15] [lint] eslint version: 8.7.0
[14:24:15] Starting subtask 'tsc'...
[14:24:15] [tsc] typescript version: 4.5.5
[14:24:22] Finished subtask 'copy-static-assets' after 8.29 s
[14:24:31] Finished subtask 'tsc' after 16 s
[14:24:54] Finished subtask 'lint' after 40 s
[14:24:54] Starting subtask 'post-copy'...
[14:24:54] Finished subtask 'post-copy' after 1.95 ms
[14:24:54] Starting subtask 'configure-webpack'...
[14:24:57] Finished subtask 'configure-webpack' after 2.82 s
[14:24:57] Starting subtask 'webpack'...
[14:25:17] Finished subtask 'webpack' after 20 s
[14:25:17] Starting subtask 'reload'...
[14:25:17] Finished subtask 'reload' after 8.66 ms

According to the tutorial, the workbench opens in my browser, but no request is fired.  Requesting the https://localhost:4231/temp/manifest.js URI by hand with the browser i receive an "Cannot GET /temp/manifest.js" error. 
I ran out of ideas what to try to make it work. Thanks for any help.
Kind regards, Tobi

**Comment by AJIXuMuK** [comment]
@Ziefle-T seems like the terminal process can't access your web browser to open a tab.
But you can always manually open workbench pages in the browser by yourself. Just copy the URL from the `serve.json` file and open it in the browser.

**Comment by Ziefle-T** [comment]
@AJIXuMuK Hi! There is a browser window opening up. But no request firing towards my localhost. At least i can't see one in my console log. Therefore my custom webpart doesn't appear in the webpart picker in the workbench.

**Comment by AJIXuMuK** [comment]
@Ziefle-T have you run `gulp trust-dev-cert` before running the project?
You need to run this command once per machine (or per node).

**Comment by Ziefle-T** [comment]
@AJIXuMuK Yes.  I've double checked all the previous steps. And trusting the dev-cert was one of them.

**Comment by Ziefle-T** [comment]
Hi @AJIXuMuK. Sorry to bother you, but do you have any more ideas? I personally am at the end of my expertise.

**Comment by AJIXuMuK** [comment]
@Ziefle-T the only thing I can imagine is some network configuration that blocks incoming requests. 
Try another machine or vm

**Comment by AJIXuMuK** [comment]
I will be closing the issue as it seems to be machine/network config specific one.
